test/data/conformance/no-financial.xml

```diff
@@ -96,6 +63,12 @@
           <eeSummary:radiationExposureInd>true</eeSummary:radiationExposureInd>
         </eeSummary:specialFactors>
       </eeSummary:enrollmentDeterminationInfo>
+      <eeSummary:financialsInfo xmlns:eeSummary="http://jaxws.webservices.esr.med.va.gov/schemas">
```

Comment:
`discloseFinancialInformation` is false, but `financialsInfo` is still present here. This is the test I sort of want to verify is behaving correctly, as referenced in my other comment.